components/webdriver_server/lib.rs

```diff
@@ -1396,15 +1399,27 @@ impl Handler {
         parameters: &JavascriptCommandParameters,
     ) -> WebDriverResult<WebDriverResponse> {
         let func_body = &parameters.script;
-        let args_string = "window.webdriverCallback";
+        let mut args = parameters.args.as_ref().map_or(vec![], |args| {
+            args.iter().map(|a| format!("{}", a)).collect()
```

Comment:
```suggestion
            args.iter().map(|a| a.to_string()).collect()
```

can also use ToString::to_string